src/ripple/app/tx/impl/Change.cpp

```diff
@@ -221,4 +235,127 @@ Change::applyFee()
     return tesSUCCESS;
 }
 
+TER
+Change::applyUNLModify()
+{
+    if (view().seq() % 256 != 0)
+    {
+        JLOG(j_.warn()) << "N-UNL: applyUNLModify, not a flag ledger, seq="
+                        << view().seq();
+        return tefFAILURE;
+    }
+
+    if (!ctx_.tx.isFieldPresent(sfUNLModifyDisabling) ||
+        ctx_.tx.getFieldU8(sfUNLModifyDisabling) > 1 ||
+        !ctx_.tx.isFieldPresent(sfLedgerSequence) ||
+        !ctx_.tx.isFieldPresent(sfUNLModifyValidator))
+    {
+        JLOG(j_.warn()) << "N-UNL: applyUNLModify, wrong Tx format.";
+        return tefFAILURE;
+    }
+
+    bool disabling = ctx_.tx.getFieldU8(sfUNLModifyDisabling);
+    auto seq = ctx_.tx.getFieldU32(sfLedgerSequence);
+    if (seq != view().seq())
+    {
+        JLOG(j_.warn()) << "N-UNL: applyUNLModify, wrong ledger seq=" << seq;
+        return tefFAILURE;
+    }
+
+    Blob validator = ctx_.tx.getFieldVL(sfUNLModifyValidator);
+    if (!publicKeyType(makeSlice(validator)))
+    {
+        JLOG(j_.warn()) << "N-UNL: applyUNLModify, bad validator key";
+        return tefFAILURE;
+    }
+
+    JLOG(j_.info()) << "N-UNL: applyUNLModify, disabling=" << disabling
+                    << " seq=" << seq
+                    << " validator data:" << strHex(validator);
+
+    auto const k = keylet::negativeUNL();
+    SLE::pointer nUnlObject = view().peek(k);
```

Comment:
This is just `std::shared_ptr<SLE>` right? Just a nit, but I prefer using the actual type here. I don't think the type alias actually gives us anything, and using the actual type helps readability.

Reply:
Right, it is `std::shared_ptr<SLE>`. But I will keep as-is if that is OK, to be consistent to existing code.

Reply:
That's fine - we can do a cleanup to fix that separately; it's easier if such "mechanical" changes are done all together so that _everything_ migrates to the new style.